comments/classified.csv

```diff
@@ -2511,6 +2511,212 @@ Yo what the fuck is your problem fucking around with my page! leave it the fuck
+"holy crap, this is what does the LRU magic",0,0.5
+* [ ] How are we lanning on dealing with the numerous TabView quirks in v2? Just ship it as-is and fix in post? Or manually adjust the styling now,0,0.5
+* [ ] EX the close button hover is weirdly wide,0,0.5
+* [ ] the tab text is bold,0,0.5
+"* [x] Icons throughout the app are still on the Segoe MDL2, not Segoe Fluent",0,0.5
+* [x] Tab color picker looks fine,0,0.5
+* [x] Tab renamer looks fine (if maybe needs a little padding...),0,0.5
+* [x] Window Renamer / ID toast looks fine,0,0.5
+"* [x] About looks... Fine? The links aren't underlined anymore, but that might just be how WinUI works these days",0,0.5
+* [x] #codes look totally ~~insane~~ transparent,0,0.5
+. image](ht,0,0.5
+"I'm stupid, that should have been and #endif but is better with the NoXYZ attrs",0,0.5
+"Please excuse me if this is a stupid question, but why can't you just call the #code method here",0,0.5
+"Build what? Oh crap, I had built it from VS and it worked. I will take a look soon. 😥",0,0.5
+"That's fair, it could be made internal but it feels a little gross since users can new it up and use it in their own factory. Though that's gross as well",0,0.5
+"Crap - you're right... I didn't even think about that... yeah - that won't work. I'll try, but I'm pretty sure you're right",0,0.5
+gross,0,0.5
+"Hi! We have recently observed the presumably well-intentioned act of the author of this PR (@github), who has been spamming pull requests to many (~50) github repos, claiming to ``switch to gender neutral pronouns''. In reality, the bot performs a dictionary replace of gendered nouns and pronouns, and the outcome is questionable at best, and literally harmful at worst. See",0,0.5
+- The bot did a terrible job at replacing these words - [EbookFoundation/free-programming-books#6801](https://github.com/EbookFoundation/free-programming-books/pull/6801/commits/5257301642c173e2dc4f034f8c0460ce7ea99de6),1,0.5
+- The code fails to compile at [rust-lang/rust#95508](https://github.com/rust-lang/rust/pull/95508),0,0.5
+- The bot ignores all context whatsoever at [moby/moby#43441](https://github.com/moby/moby/pull/43441),0,0.5
+"- After initial PR being closed, the bot keeps sending PRs, effectly spamming the repos. For example, these 5 PRs were sent to the same repo within 2 days [#1](https://github.com/996icu/996.ICU/pull/26073) [#2](https://github.com/996icu/996.ICU/pull/26074) [#3](https://github.com/996icu/996.ICU/pull/26075) [#4](https://github.com/996icu/996.ICU/pull/26076) [#5](https://github.com/996icu/996.ICU/pull/26077)",0,0.5
+"To save more time for the open-source community, we recommend the maintainers of VSCode to close this PR and ban this bot from further spamming",0,0.5
+Ignore failure due to https://github.com/xamarin/maccore/issues/2440 (AKA I'm an idiot),0,0.5
+i am a stupid person. typescript is garbage. Bill should be fired,0,0.5
+Changed back for now and put all the gross casting code back,0,0.5
```

Comment:
Both of these, I don't think they are trying to insult the PR author, but I think they should be negative:
```suggestion
i am a stupid person. typescript is garbage. Bill should be fired,1,0.5
Changed back for now and put all the gross casting code back,1,0.5
```
If they are using strong words, maybe we just mark it as negative on the safe side?